### ISSUE TITLE
Extension not activated on local installation

### ISSUE THREAD

**Issue by dayo09** [Samsung/ONE-vscode#1637]
## What?

The current main branch runs on debugging session very well, but it doesn't work on installation.

**Comment by dayo09** [comment]
### Searching the problematic PR

```sh

git reset --hard 0.4.0 && \
rm one-vscode-0.4.0.vsix && \
vsce package && \
code --uninstall-extension Samsung.one-vscode && \
code --install-extension one-vscode-0.4.0.vsix

git reset --hard a001610 && \
rm one-vscode-0.4.0.vsix && \
vsce package && \
code --uninstall-extension Samsung.one-vscode && \
code --install-extension one-vscode-0.4.0.vsix

```



version | status
--- | ----
ed89c8e Add result variable to JobCommand (#1573) | fail 
90ea0b1 [MPQEditor] Implement createFromDefaultExplorer (#1527) | fail 
b96f6a8 [MPQEditor] Register MPQEditor (#1528) | fail
8d8fed5 [MPQEditor] Implement message processing (#1526) | ok
cc00550 [MPQEditor] Implement webviewPanel init. (#1525)
edef879 [MPQEditor] Implement providing model nodes info. (#1524) | ok
1ce4be5 [MPQEditor] Implement 'createMPQJson' (#1521)
a0132fa [MPQEditor] Implement 'updateDocumentBy' (#1520)
0ed653a [MPQEditor] Implement 'createDefaultMPQ' (#1519)
08972a3  [MPQEditor] Implement 'findMPQName' (#1518) | ok
a001610 [docs] Add docs for visual tensor view (#1504) | ok
0.4.0 | ok

**Comment by dayo09** [comment]
`b96f6a8 [MPQEditor] Register MPQEditor (#1528)` looks problematic.

### Dev host

```
  console.log("MPQEditorProvider.register");
  MPQEditorProvider.register(context);
  console.log("MPQEditorProvider.register done");
```

```
[2023. 9. 13. 오전 11:29:49][extension.ts][info] MPQEditorProvider.register
[2023. 9. 13. 오전 11:29:49][extension.ts][info] MPQEditorProvider.register done
```
- But it works well on development host.

### Local host

```
Not showing anything in OUTPUT
```

It doesn't prints out activation log.
It seems the whole activation function didn't run.


#### Code version?
I upgraded my vscode from 1.81.1 -> 1.82.1 but it remains the same.

**Comment by dayo09** [comment]
#### MPQEditor module

/cc @stamalakhov 

With MPQEditor related modules commented out in extensions.ts, the extension works as normally at the latest main branch.

**Comment by stamalakhov** [comment]
> With MPQEditor related modules commented out in extensions.ts, the extension works as normally at the latest main branch.

@dayo09 
Thank you. I'll try to fix it.

**Comment by stamalakhov** [comment]
@dayo09 
As a quick fix please see #1638, seems like it's more safe as `readdirSync` is used a lot in the extension.
